lib/expand/fsm.go

```diff
@@ -146,7 +146,17 @@ type fsmEngine struct {
 
 // GetExecutor returns a new executor based on the provided parameters
 func (e *fsmEngine) GetExecutor(p fsm.ExecutorParams, remote fsm.Remote) (fsm.PhaseExecutor, error) {
-	return e.Spec(p, remote)
+	logger := &fsm.Logger{
```

Comment:
This will hopefully include more error information in the operator logs if the phase fails to initialize.